gitlab-runner-17.1.yaml

```diff
@@ -27,15 +30,22 @@ pipeline:
     with:
       deps: github.com/Azure/azure-sdk-for-go/sdk/azidentity@v1.6.0 github.com/hashicorp/go-retryablehttp@v0.7.7
 
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/
+      cp dockerfiles/runner/apline/entrypoint "${{targets.destdir}}"/entrypoint
```

Comment:
typo, ~apline~ alpine.

Reply:
Yeah, embarassing. Fixed it, did it in a hurry